listenstore/listenstore/utils.py

```diff
@@ -3,23 +3,32 @@
 import argparse
 from setproctitle import setproctitle
 import json
-
-
-## TODO read config from whereever
-def config():
-    return {
-        "kafka_server": "localhost:9092",
-        "cassandra_server": "localhost",
-        "cassandra_keyspace": "listenbrainz"
-    }
+import ConfigParser
+import os
+
+def config(opt_vars):
+    """ Parse config file else return default values """
```

Comment:
You can load configuration values from config file written in regular Python using this https://github.com/pallets/flask/blob/master/flask/config.py#L111-L137. Can just reuse that class since Flask is one of current requirements.